backend/sftp/sftp.go

```diff
@@ -327,7 +339,47 @@ and pass variables with spaces in in quotes, eg
 
 `,
 			Advanced: true,
-		}},
+		}, {
+                        Name:    "user_defined_ciphers",
+                        Default: fs.SpaceSepList{},
+                        Help: `Space separated list of ciphers to be used for session encryption, ordered by preference.
+
+At least one must match with server configuration. This can be checked for example using ssh -Q cipher.
+
+This must not be set if use_insecure_cipher is true.
+
+Example:
+
+    aes128-ctr aes192-ctr aes256-ctr aes128-gcm@openssh.com aes256-gcm@openssh.com
+`,
+                        Advanced: true,
+                },{
+                        Name:    "user_defined_key_exchange",
```

Comment:
Can you remove the user_defined_ prefixes here. They will make the config options very long and they find add anything I think.